styleguide/derek/incubation/homepage.ejs

```diff
@@ -5,6 +5,9 @@
   'body': 'Redhat names Rackspace North America Managed Service Partner of the Year'
 }) %>
 
+<!-- Case Study -->
```

Comment:
Will you move this comment inside the partial file, and then maybe rename it "Customer Story River"?